report/generator/report-generator.js

```diff
@@ -7,6 +7,9 @@
 
 const htmlReportAssets = require('./report-assets.js');
 
+/** @typedef {import('../../types/lhr/lhr').default} LHResult */
+/** @typedef {import('../../types/lhr/flow').default} FlowResult */
```

Comment:
we could make a globals `d.ts` file for the report generator too, but it didn't seem worth it for just these two types. I don't feel strongly either way, though